sources/packages/api/src/route-controllers/education-program/models/save-education-program.dto.ts

```diff
@@ -1,5 +1,8 @@
 import { ProgramIntensity } from "../../../database/entities/program-intensity.type";
 
+/**
+ * Dto that represents education programs form object.
+ */
 export interface EducationProgramDto {
   name: string;
   description: string;
```

Comment:
I see the the related model has been changes to have description as optional, can we implement this here?
description?: string;

Reply:
Good catch.